freshmaker/logger.py

```diff
@@ -139,24 +127,35 @@ def init_logging(conf):
     """
     Initializes logging according to configuration file.
     """
-    log_format = '%(levelname)s - %(message)s'
+    log = logging.getLogger()
+    log.setLevel(conf.log_level)
```

Comment:
I just realized we should not change the default log level of root logger, instead of that, the `conf.log_level` should be applied to `freshmaker` logger.

Reply:
Done!